### DISCUSSION TITLE
Request for Localization Feature in spacedrive

### DISCUSSION THREAD

**Discussion by Arthur-AKun** [spacedriveapp/spacedrive#1867] [Feature Request]
Hello, developer of SpaceDrive,

I hope this message finds you well. I am writing to bring your attention to a feature that I believe would greatly enhance the user experience of this software - localisation.

As an international user, I have found that the current lack of a localised interface makes it difficult for me to fully utilise the software's capabilities. While the core functionality is impressive, the absence of language support limits its accessibility and usability for non-English speaking users.

Localisation would not only make the software more inclusive but also increase its potential user base. By providing translations for all text elements, users from different regions would be able to navigate and understand the software with ease. This would greatly improve the overall user experience and satisfaction.

I understand that implementing localisation may require additional resources and effort, but I firmly believe that the benefits outweigh the costs. A localised interface would not only attract a wider audience but also demonstrate the software's commitment to inclusivity and global reach.

Thank you for considering my suggestion. I look forward to seeing this feature implemented in the future updates of the software.

**Comment by utkubakir** [comment]
We will ship this very soon

**Reply by utkubakir** [reply]
Shipped :)
